packages/api/src/hooks/p2p/entity/order-dispute/index.ts

```diff
@@ -0,0 +1 @@
+export * as orderDispute from './p2p-order-dispute';
```

Comment:
Wait shouldn't this be inside `order` entity?

Reply:
Like its part of order entity, soo maybe like:
```
hooks
   |-p2p
       |-entity
          |-payment-method
          |-order
               |-useOrderDispute
```

Reply:
yeah.. even i thought so.. but i had discussed with @farrah-deriv .. so dispute,chat  and review are different entities which come in the same level as order..

Reply:
Inside hooks/p2p/entity/index.ts
```
export * from './payment-method';
export * from './order'
```

Reply:
Then we can use the hook like:
```
p2p.order.useDispute()
```

Reply:
I believe dispute is a different entity. It's a complaint made to an order which has its own details (the reason, complainer, and the order_id).